.gitmodules

```diff
@@ -22,3 +22,6 @@
 [submodule "themes/nc-ionos-theme"]
 	path = themes/nc-ionos-theme
 	url = git@github.com:IONOS-Productivity/nc-ionos-theme.git
+[submodule "apps-custom/nc-vue-material-design-icons"]
```

Comment:
This is no app. How about putting it under `/built-npms/nc-vue-material-design-icons` ?

Reply:
I favor not to checkout the repo as submodule.. 
...instead build a npm package and reference it in package.json (because we have to rebuild all apps with new icons).

Reply:
I see. Have you tested creating an NPM release in GitHub and referencing in GitHub already?

Reply:
> I see. Have you tested creating an NPM release in GitHub and referencing in GitHub already?

no test as for now yet.. ..if we want to follow this approach will gladly try it out https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry

Reply:
moved to `custom-npms/nc-vue-material-design-icons`